src/context/AppProvider/hooks/useAccountsFetchQuery.tsx

```diff
@@ -139,34 +133,39 @@ export const useAccountsFetchQuery = () => {
                 return account.accounts.byId[accountId].hasActivity
               })
 
-          // don't add accounts with no activity past account 0
+          // If account has no activity and it's not account 0, stop checking this chain
```

Comment:
nitpick: find the previous comment more better nicer, i.e "past account 0" reads nicer

Reply:
Reverted in [0917dd0](https://github.com/shapeshift/web/pull/9727/commits/0917dd05ff9b6c3553e43ff082b3915fe9081dee)